tests/e2e/testsLibrary/LanguageServerTests.ts

```diff
@@ -167,6 +167,10 @@ export class LanguageServerTests {
 
         test('Run debug', async () => {
             await this.debugView.clickOnDebugConfigurationDropDown();
+            // workaround for debug configuration not present
+            await this.debugView.clickOnDebugConfigurationItem('Add Configuration...');
+            await this.debugView.clickOnDebugConfigurationDropDown();
+            // ===
```

Comment:
In my eyes, this is extra. I would suggest to leave just empty row.

Reply:
This is the way the fix was implemented in your PR, instead of doing the refresh and it fixed our problems when I tried it from a custom-built dockerimage